HW_1/zadanie2.py

```diff
@@ -0,0 +1,14 @@
+#Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+#Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+#*Пример:*
+#6 -> 1  4  1
+#24 -> 4  16  4
+#    60 -> 10  40  10
+
+peta = int(input('Введите какое количество сделал Петя: ')) 
+sergei = int(input('Введите какое количество сделал Сергей:  '))
+kata=(peta+sergei)*2
+print('У Кати получилось журавликов: ',kata)
+itog=kata+sergei+ peta
```

Comment:
Транслитерация кириллических конструкций не используется ни только в Python, но и в большинстве других языков.
```suggestion
itog = kata + sergei + peta
```